webpack.config.js

```diff
@@ -84,6 +85,8 @@ module.exports = (params = {}) => {
           NODE_ENV: JSON.stringify(env),
         },
       }),
+
+      new UnminifiedWebpackPlugin()
```

Comment:
As I see it, there is no need for this plugin here.
There is a command `build:all`, which sequentially launches `npm run build` and `npm run build:production`.
I got this at the end:
<img width="287" alt="Screenshot 2023-08-24 at 11 43 40" src="https://github.com/JetBrains/kotlin-playground/assets/3338311/48c9c404-a6bb-4962-a37d-bf4b0d4d54fd">

Reply:
You're right, will remove